lib/features/context-pad/ContextPad.js

```diff
@@ -622,25 +626,35 @@ ContextPad.prototype._updatePosition = function() {
  * using the `djs-element-hidden` or `djs-label-hidden` markers.
  */
 ContextPad.prototype._updateVisibility = function() {
-  if (!this.isOpen()) {
-    return;
-  }
 
-  var self = this;
+  const updateFn = () => {
 
-  var target = this._current.target;
+    this._requestedFrame = null;
```

Comment:
It looks that it's not used anymore.

```suggestion
```

Reply:
You are right, I missed it when refactoring 👍